### DISCUSSION TITLE
no funciona el script hay comandos que dan error

### DISCUSSION THREAD

**Discussion by jorge4684** [josuapsianturi/velflix#145] [General]
no funciona el script hay comandos que dan error

**Comment by josuapsianturi** [comment]
Sorry for this delayed reply!! 

I already fix it here https://github.com/josuapsianturi/velflix/commit/8b560531219d19589ae8f6d3c30a0a29a8c33aa2 , 85045f1071cfa0ca508016fc9e352a726597aae9

Thanks
